File: t-bank/test-tasks/task-2.js
const readline = require("readline");

const rl = readline.createInterface({
    input: process.stdin,
    output: process.stdout
});

async function solve() {
    const input = await readLine();
    const parts = parseInt(input.trim());
    let curr = 1;
    let i = 0;

    while (curr < parts) {
        curr *= 2;
        i += 1;
    }

    console.log(i);
    rl.close();
}

function readLine() {
    return new Promise((res) => {
        rl.once("line", res);
    });
}

solve();

File: t-bank/test-tasks/task-2.py
# 1 <= N <= 2 * 10^9

# Binary tree
parts = int(input())
queue = [parts]
res = 0
while queue:
    for _ in range(len(queue)):
        part = queue.pop()
        part //= 2
        if part:
            queue.append(part)
    res += 1
print(res)


# n = int(input())
# k = 0
# curr = 1
# while curr < n:
#     curr *= 2
#     k += 1
# print(k)
